src/inspirehep-holdingpen-js/templates/inspirehep-holdingpen-js/details/details_decision_hep.html

```diff
@@ -10,7 +10,7 @@
   </p>
 
   <div
-    ng-if="vm.record._extra_data._action == 'hep_approval' && !vm.ingestion_complete">
```

Comment:
Removed the `!vm.ingestion_complete` to make the if conditions similar to those in result.html

Reply:
could you please check what ``ingestion_complete`` checks?

Reply:
is it used on the results?